src/Symfony/Component/Messenger/Tests/WorkerTest.php

```diff
@@ -245,6 +247,37 @@ public function testWorkerWithMultipleReceivers()
         $this->assertSame([$envelope1, $envelope2, $envelope3, $envelope4, $envelope5, $envelope6], $processedEnvelopes);
     }
 
+    public function testWorkerLimitQueues()
+    {
+        $receiver1 = $this->createMock(QueueReceiverInterface::class);
```

Comment:
This is still named `$receiver1`

Reply:
you are too quick, i did not push yet ;-)